packages/vuepress-theme-nimatron/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@okta/vuepress-theme-nimatron",
+  "version": "0.3.0",
+  "description": "VuePress theme for Odyssey Documentation",
```

Comment:
Let's mark this is a `private` package so we don't publish it to our internal or public artifactory. This should just a local devDependency for our doc site.

Reply:
Good call here, fixing.